jdi-light/src/main/java/com/epam/jdi/light/logger/JDILogger.java

```diff
@@ -61,7 +63,7 @@ public void setLogLevel(LogLevels level) {
         setRootLevel(getLog4j2Level(level));
         setLevel(name, getLog4j2Level(level));
     }
-    private Safe<Integer> logOffDeepness = new Safe<>(0);
+    private final Safe<Integer> logOffDeepness = new Safe<>(0);
```

Comment:
Codacy found an issue: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/gh/jdi-testing/jdi-light/pullRequest?prid=6346938)